src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml

```diff
@@ -162,3 +168,36 @@ mb:
         description: Dimensions-Handler gruppiert und steuert in Der Karte Source Instanzen
     dimensionsets:
       no_dimension: Keine Dimension vorhanden.
+  core:
+    dimensionset:
+      admin:
+        title: Titel
+        group: Gruppe
+        extent: Ausmaß
+    dimensionshandler:
+      admin:
+        tooltip: Tooltip
+        dimensionsets: Dimensionssets
+    wmsloader:
+      admin:
+        defaultformat: Standardformat
+        defaultinfoformat: Standard-Infoformat
+    admin:
+      ? ''
+      : vendorspecifictype:
+          name: Namen
```

Comment:
: vendorspecifictype: -> can not be correct : before 

Name nicht Namen, oder?